### ISSUE TITLE
Installing microk8s enable openebs -> jivavolume-controller: no matches for kind "PodDisruptionBudget" in version "policy/v1beta1"

### ISSUE THREAD

**Issue by keskival** [canonical/microk8s#3639]
#### Summary
Installing OpenEBS Jiva with `microk8s enable openebs` seems to work but pods created with mounted JivaVolumes will hang forever in ContainerCreating state.

Investigating further shows that an old PodDisruptionBudget apiVersion is in use:
```
$ kubectl describe JivaVolumes -A | grep PodDisr
  Warning  Bootstrap  4m46s (x1750 over 4h49m)  jivavolume-controller  failed to bootstrap volume, due to error: failed to get the pod disruption budget details: pvc-e85d22c7-8a0d-4660-a47f-faf5eb4ec880-pdb: no matches for kind "PodDisruptionBudget" in version "policy/v1beta1"
```

Installing OpenEBS Jiva with their official Helm charts from here works: https://github.com/openebs/jiva-operator/blob/develop/docs/quickstart.md

#### What Should Happen Instead?
The OpenEBS Jiva volumes should be mounted successfully.

#### Reproduction Steps

```
$ microk8s version
MicroK8s v1.25.4 revision 4221
```

1. `microk8s enable community`
2. `microk8s enable openebs`
3. Install these resources:
```
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  finalizers:
  - kubernetes.io/pvc-protection
  labels:
  name: jivaclaim
  namespace: jivatest
spec:
  accessModes:
  - ReadWriteOnce
  resources:
    requests:
      storage: 1Gi
  storageClassName: openebs-jiva-csi-default
  volumeMode: Filesystem
---
apiVersion: v1
kind: Pod
metadata:
  name: jivatest
  namespace: jivatest
spec:
  volumes:
  - name: jiva
    persistentVolumeClaim:
      claimName: jiva-claim
  containers:
  - name: datacopy
    image: ubuntu
    command:
    - "sleep"
    - "360000000"
    volumeMounts:
    - mountPath: "/mnt/jiva"
      name: jiva
```

Note that the pod never successfully comes up.

#### Introspection Report
[inspection-report-20221223_070816.tar.gz](https://github.com/canonical/microk8s/files/10292441/inspection-report-20221223_070816.tar.gz)

#### Can you suggest a fix?
Update the chart from upstream.

#### Are you interested in contributing with a fix?
Not right now.

**Comment by keskival** [comment]
This relates to this previous bug in upstream: https://github.com/kyverno/kyverno/issues/2983

**Comment by neoaggelos** [comment]
I believe a backport of https://github.com/canonical/microk8s-community-addons/pull/103 is needed to the 1.25 branch for this.

**Comment by ktsakalozos** [comment]
Thank you for reporting this @keskival we have backported the fix to 1.25 and it will be available with the next 1.25 patch release (make sure you `microk8s repo update community` to get the this fix).

**Comment by turosretes** [comment]
Hi,
When will it be usable? Is there an estimation?
Thank you in advance
